docs/en/sql-reference/window-functions/index.md

```diff
@@ -753,31 +754,30 @@ All GitHub issues related to window functions have the [comp-window-functions](h
 
 These tests contain the examples of the currently supported grammar:
 
-https://github.com/ClickHouse/ClickHouse/blob/master/tests/performance/window_functions.xml
+<https://github.com/ClickHouse/ClickHouse/blob/master/tests/performance/window_functions.xml>
 
-https://github.com/ClickHouse/ClickHouse/blob/master/tests/queries/0_stateless/01591_window_functions.sql
+<https://github.com/ClickHouse/ClickHouse/blob/master/tests/queries/0_stateless/01591_window_functions.sql>
 
 ### Postgres Docs {#postgres-docs}
 
-https://www.postgresql.org/docs/current/sql-select.html#SQL-WINDOW
+<https://www.postgresql.org/docs/current/sql-select.html#SQL-WINDOW>
 
-https://www.postgresql.org/docs/devel/sql-expressions.html#SYNTAX-WINDOW-FUNCTIONS
+<https://www.postgresql.org/docs/devel/sql-expressions.html#SYNTAX-WINDOW-FUNCTIONS>
 
-https://www.postgresql.org/docs/devel/functions-window.html
+<https://www.postgresql.org/docs/devel/functions-window.html>
 
-https://www.postgresql.org/docs/devel/tutorial-window.html
+<https://www.postgresql.org/docs/devel/tutorial-window.html>
```

Comment:
@dhtclk these break the build because `<` is a valid character in JSX. Let's manually fix these by keeping the list and using markdown link syntax.